server/project/test/ping_handler_test.py

```diff
@@ -1,4 +1,4 @@
-from test.test_base import TestBase
+from project.test.test_base import TestBase
 import json
```

Comment:
If we are using flask-rest+ we shouldn't need json.

Reply:
Noted. But going to leave this for now as we have not added that into the project yet and it seems   to be needed currently.